specification/metadata/schema/scale.md

```diff
@@ -0,0 +1,17 @@
+# Scale
```

Comment:
Maybe Number Scale, to be clear and explicit?

Reply:
I thought "precision" was the number of decimal places. Scale sounds like ones vs. thousands vs millions.

Reply:
I had to check it also.

"Precision is the number of digits in a number. Scale is the number of digits to the right of the decimal point in a number" - https://learn.microsoft.com/en-us/sql/t-sql/data-types/precision-scale-and-length-transact-sql?view=sql-server-ver16